document/CoreFormat.mdk

```diff
@@ -508,6 +552,43 @@ whose value is a string specifying that path.
 If the location of the file within its containing property cannot be expressed by means of a path,
 then the `pathFromParent` property shall be absent.
 
+If the `file` object occurs as the only member of an array within the `run.files` object (&sect;[#run-files]),
+then the `pathFromParent` property may be absent.
```

Comment:
yes

---
In reply to: [60333407](https://github.com/sarif-standard/sarif-spec/pull/161#discussion_r60333407) [](ancestors = 60333407)